app/task_routes.py

```diff
@@ -0,0 +1,108 @@
+from flask import Blueprint, request, jsonify, make_response, abort
+from datetime import datetime
+from app.models.task import Task
+from app import db
+from .helper_functions import validate_model
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
load_dotenv() only needs to be called on startup to load the values in .env

After it's executed from create_app() we only need to import os and get the key from os in task_routes.py